Project/Assets/ML-Agents/Examples/Crawler/Scripts/CrawlerAgent.cs

```diff
@@ -98,10 +109,11 @@ public void CollectObservationBodyPart(BodyPart bp, VectorSensor sensor)
     /// </summary>
     public override void CollectObservations(VectorSensor sensor)
     {
-        sensor.AddObservation(Quaternion.FromToRotation(body.forward, orientationCube.transform.forward));
-
+        sensor.AddObservation(body.rotation);
```

Comment:
Why did this observation change?

Reply:
this was an oversight from some testing i was doing. i changed it back to the previous observations.